pages/styles.css

```
@@ -0,0 +1,372 @@
+    box-shadow: 2px 2px 8px #0C0C0C;
+    color: #FFF;
+    text-transform: uppercase;
+    font-size: 10px;
+    border: none;
+    border-radius: 25px;
+    padding:10px 20px;
+}
+
+h1, h2, h3{
+    font-family: 'Merriweather Sans', sans-serif;
+}
+
+a{
+    color: var(--main-bgcolor);
+    text-decoration: none;
+}
+
+:root{
+    --gradient-bgcolor: linear-gradient(90deg, rgba(72,69,228,1) 0%, rgba(87,34,205,1) 100%);
+    --main-bgcolor: #5722CD;
+    --button-bgcolor: #ED4385;
+    --project-bgcolor: #F3F6F9;
+    --tekstkleur: #4845E4;
+    --ul-etc: #F2F0FF;
+    --stan-text: #0C0C0C;
+    --servicetegel: #7A7A7A;
+}
+
+/* ------------------------------- 3. Areas ---------------------------------------- */
+/* --------------- 3.1 Header ------------------------- */
+
+header{
+    background: linear-gradient(90deg, rgba(72,69,228,1) 0%, rgba(87,34,205,1) 100%);
+    color: #FFF;
+    position: relative;
+}
+
+nav{
+    display: flex;
+    justify-content: flex-end;
+    width: 100%;
+    margin-bottom: 35px;
+}
+
+ul{
+    display: flex;
+    gap: 35px;
+    list-style: none;
+    padding: 5px 0 10px;
+    color: var(--ul-etc);
+}
+
+nav li a{
+    color: #FFF;
+}
+
+.header-text{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    row-gap: 30px;
+    max-width: 600px;
+    margin-bottom: 150px;
+}
+
+.drawing {
+    width: 40px;
+    position: absolute;
+    bottom: 0;
+    left:calc(50% - 20px);
+}
+
+.triangle {
+    border-right: 100vw solid transparent;
+    border-bottom: 50px solid white;
+    position: absolute;
+    left: 0;
+    bottom: 0;
+}
+
+/* --------------- 3.2 Introduction ------------------- */
+
+main{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100vw;
+}
+
+.introduction{
+    width: 100%;
+    background-color: #FFF;
+    color: var(--main-bgcolor);
+}
+
+.introduction-text{
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    flex-wrap: wrap;
+    align-content: center;
+    text-align: center;
+    max-width: 600px;
+    margin: 50px 0;
+}
+
+.introduction-small-text{
+    font-size: 14px;
+    font-weight: bold;
+    padding: 50px 0 0;
+    color: var(--stan-text);
+}
+
+/* --------------- 3.3 Image area ------------------- */
+
+#image-area{
+    background-color: var(--project-bgcolor);
+    width: 100vw;
+    position: relative;
+}
+
+.middle-driehoek-top{
+    border-left: 100vw solid transparent;
+    border-top: 50px solid white;
+    position: absolute;
+    left: 0;
+    top: 0;
+}
+
+.middle-driehoek-bottom{
+    border-right: 100vw solid transparent;
+    border-bottom: 50px solid white;
+    position: absolute;
+    left: 0;
+    bottom: 0;
+}
+
+.image-article{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin: 75px 0;
+    flex-wrap: wrap-reverse;
```

Ik had hier een nowrap van gemaakt zodat bij het smaller maken van de pagina het plaatje niet boven de tekst komt
```suggestion
    flex-wrap: nowrap;
```